server/package.json

```diff
@@ -26,9 +26,7 @@
     "web-tree-sitter": "^0.20.8"
   },
   "scripts": {
-    "prepack": "../scripts/fetch-docs.sh && tsc --outDir out && cp -r ./resources ./tree-sitter-bitbake.wasm out/",
-    "installServer": "installServerIntoExtension ../client ./package.json ./tsconfig.json && npm run installServerResources",
-    "installServerResources": "cp -r ./resources ./tree-sitter-bitbake.wasm ../client/server/"
+    "prepack": "cd .. && npm run compile"
```

Comment:
Getting command not found when starting language-server-bitbake for vim
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/f9271354-8310-4f23-aafe-ecd28b2966e4)
To reproduce, 
 - cd server
 - npm pack
 - npm install -g ./*.gz
 - language-server-bitbake --stdio (should start and await input)

Reply:
I think once again you forgot to update the server's package.json bin location (out/server.js instead of out/src/server.js)

Reply:
Make sure to also test that the resources paths are all correct (docs and .wasm) once packaged for vim and VSCode.

Reply:
Yes, my bad. `bin` updated.

Reply:
Now working on NVIM